requirements.txt

```diff
@@ -1,5 +1,6 @@
 # Copyright 2021 Canonical Ltd.
 # See LICENSE file for licensing details.
+cosl
```

Comment:
Might as well add it to PYDEPS so we don't forget when we remove it from here.

Reply:
It's already there :)